packages/terra-worklist-data-grid/src/subcomponents/Row.jsx

```diff
@@ -159,7 +159,10 @@ function Row(props) {
 
   return (
     <tr
-      className={cx([isSelected ? 'row-selected' : 'worklist-data-grid-row', theme.className])}
+      className={cx('worklist-data-grid-row', {
+        'row-selected': isSelected,
+        selectable: hasRowSelection,
```

Comment:
You can't use only the hasRowSelection prop.  You also have to ensure that the row is selectable.  There should be an isSelectable prop on the rowShape.  That needs to be added so that consumers can specify whether a row is selectable.

Reply:
As we discussed offline, @eawww will follow up on this and let us know if its required.